diffsync/__init__.py

```diff
@@ -663,6 +714,34 @@ def get_by_uids(
         """
         return self.store.get_by_uids(uids=uids, model=obj)
 
+    def get_tree_traversal(self, as_dict: bool = False) -> Any:
+        """Get a string describing the tree traversal for the diffsync object.
+
+        Args:
+            as_dict (bool): Whether or not to return as a dictionary
+
+        Returns:
+            Any: A string or dictionary representation of tree
+        """
+        value_order = self._get_value_order()
+        output_dict: Dict = {}
+        for key in value_order:
+            model_obj = getattr(self, key)
+            if not get_path(output_dict, key):
+                set_key(output_dict, [key])
+            if hasattr(model_obj, "_children"):
+                children = getattr(model_obj, "_children")
+                for item in list(children.keys()):
+                    path = get_path(output_dict, key)
+                    if not path:
+                        set_key(output_dict, [key, item])
+                    else:
+                        path.append(item)
+                        set_key(output_dict, path)
```

Comment:
Perhaps?

```suggestion
                for child_key in list(children.keys()):
                    path = get_path(output_dict, key) or [key]
                    path.append(child_key)
                    set_key(output_dict, path)
```

Reply:
Let me test, not sure if get_path returns a None or not. Likely just copy this since the workflow is difficult with suggestions to selectively get